src/main/java/seedu/address/model/buyer/Name.java

```diff
@@ -10,13 +10,15 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only be in English and can contain certain special characters like hyphens and periods. "
+                    + "It should not be blank";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[a-zA-Z \\-.']*";
```

Comment:
```suggestion
    public static final String VALIDATION_REGEX = "[a-zA-Z \-.']*";
```
Tested the regex, seemed to work by removing extra escape character? Or is this as intended

Reply:
```suggestion
    public static final String VALIDATION_REGEX = "[a-zA-Z \\-.']+";
```
Can also change to `+` to match 1 or more in character set so it won't be empty